tests/unit/ApprovalType.spec.ts

```diff
@@ -61,10 +65,13 @@ describe('Initialize ApprovalType component', () => {
 
   it('component emits events when court order number entered', async () => {
     const wrapper: Wrapper<ApprovalType> = createDefaultComponent()
+    // Select court order radio to show the court order number text-field
+    const radio = wrapper.find('#court-order-radio')
+    radio.setChecked(true)
+    await Vue.nextTick()
     // Input text into text-field
     const input = wrapper.find('#court-order-number-input')
     input.setValue('89123456')
```

Comment:
I think you should await setValue. I think it's async, and you may need to wait for reactivity to complete, including validation and events being emitted.

This unit test _may_ seem to be working because you're getting the expected events below, but it's possible those are emissions from a previous event -- before popping the last event, you could verify that the length of the emitted events is 1 greater than before.